ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -806,7 +810,15 @@
         ;; Route listing with number of changes
         "Taulukossa on listattu valitussa palvelussa havaittuja muutoksia. Voit valita listalta yhden reitin kerrallaan tarkasteluun. Valitun reitin reitti- ja aikataulutiedot näytetään taulukon alapuolella kalenterissa, kartalla, vuorolistalla ja pysäkkiaikataululistalla."
 
-        [route-changes e! (:gtfs/route-changes changes) selected-route route-hash-id-type]]]
+        [:div.row
+         [:div.col-md-6 ""]
+         [:div.col-md-6 {:style {:margin-top "10px"}}
+          [form-fields/field {:label "Näytä myös reitit, joihin ei ole tulossa muutoksia"
```

Comment:
Localization :) 
This could use just FI language for all if any localization delays are to be avoided, main point is to have the string localization mechanism in place for straightforward maintenance later on. :)

Reply:
Translations added